uaclient/status.py

```diff
@@ -245,6 +245,9 @@ class UserFacingStatus(enum.Enum):
 MESSAGE_NO_ACTIVE_OPERATIONS = """No Ubuntu Advantage operations are running"""
 MESSAGE_LOCK_HELD = """Operation in progress: {lock_holder} (pid:{pid})"""
 PROMPT_YES_NO = """Are you sure? (y/N) """
+MESSAGE_REBOOT_SCRIPT_FAILED = (
+    "Failed running reboot_cmds script.\nSee: /var/log/ubuntu-advantage.log"
```

Comment:
Probably drop the newline here because this can easily fit on one-line in the NOTICES output.
```suggestion
    "Failed running reboot_cmds script. See: /var/log/ubuntu-advantage.log"
```